ktor-http/common/src/io/ktor/http/URLBuilder.kt

```diff
@@ -113,7 +113,8 @@ internal expect val URLBuilder.Companion.originHost: String?
 /**
  * Create a copy of this builder. Modifications in a copy is not reflected in the original instance and vise-versa.
  */
-public fun URLBuilder.clone(): URLBuilder = URLBuilder().takeFrom(this)
+@Deprecated("Use generated copy function instead", ReplaceWith("copy()"))
+public fun URLBuilder.clone(): URLBuilder = copy()
```

Comment:
If we decide to migrate to `copy()`, we need to ensure that it does exactly the same as `takeFrom()` and fix all usages and tests.